src/main/scala/nl.knaw.dans.easy.validatebag/rules/metadata/package.scala

```diff
@@ -246,8 +246,12 @@ package object metadata extends DebugEnhancedLogging {
     trace(())
     t.tryFilesXml.map {
       files =>
-        val nonFiles = (files \ "_").filterNot(_.label == "file")
-        if (nonFiles.nonEmpty) fail(s"files.xml: children of document element must only be 'file'. Found non-file elements: ${ nonFiles.mkString(", ") }")
+        if (files.namespace == filesXmlNamespace) {
```

Comment:
please adhere to the code style, even if IntelliJ cannot check certain things.
example: the body of the `if (files.namespace ...)` is only one line, so no `{...}` are needed.
other example: for readability put the `fail` after `if (nonFiles.nonEmpty)` on a new line.

same on changes below and/or in the rest of the code.

Reply:
Actually, as I remember it, we decided to always use braces, even when not needed. If I am not mistaken, it was even at your request ;) 

I am all for a uniform coding style, but when did we decide on this? Is this documented somewhere?

Reply:
I'm not sure. Can't remember it was my request, but actually I can't remember making up that rule with if-without-`{...}` either. Kinda confused.
Maybe it's new insights.

And I have no clue if this is documented somewhere or if IntelliJ is actually able to remove those unnecessary `{...}` in any way.

Let's leave it for now ;-)